declarative-pipeline-migration-assistant/pom.xml

```diff
@@ -6,10 +6,10 @@
     <artifactId>declarative-pipeline-migration-assistant-parent</artifactId>
     <version>1.0.0-SNAPSHOT</version>
   </parent>
-  <artifactId>declarative-pipeline-migration-assistant-plugin</artifactId>
+  <artifactId>declarative-pipeline-migration-assistant</artifactId>
   <version>1.0.0-SNAPSHOT</version>
   <packaging>hpi</packaging>
-  <name>Declarative Pipeline Migration Assistant :: Plugin</name>
+  <name>Declarative Pipeline Migration Assistant</name>
```

Comment:
Both this and the API plugin are plugins, so it was confusing to me that this one had `Plugin` in the name, but not the other one, and the naming conventions discourage including `Plugin` or `Jenkins` in the name, see https://jenkins.io/doc/developer/publishing/style-guides/#name.